src/targets/browser/index.jsx

```diff
@@ -64,15 +64,6 @@ document.addEventListener('DOMContentLoaded', () => {
     token: data.cozyToken
   })
 
-  cozy.bar.init({
-    appEditor: data.cozyAppEditor,
-    appName: data.cozyAppName,
-    cozyClient,
-    iconPath: data.cozyIconPath,
-    lang: data.cozyLocale,
-    replaceTitleOnMobile: false
-  })
-
```

Comment:
why?

Reply:
Oh I  didn't see the removal of https://github.com/cozy/cozy-home/pull/1369/files#diff-939604599489c47d31ecd9f9504fbf78L17 thanks!